simpleflow/base.py

```diff
@@ -0,0 +1,6 @@
+class Submittable(object):
```

Comment:
Eh, but not so useful as I expected...

Immediate problem: Our code submits `simpleflow.*Task`s instances which must be converted to `simpleflow.swf.*Task`s...